firestore.rules

```diff
@@ -2,15 +2,27 @@ rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
     match /{document=**} {
-      allow read, write: if
-          request.time < timestamp.date(2021, 5, 1);
+      allow read, write: if request.auth != null && get(/databases/$(database)/documents/users/$(request.auth.uid)).data.isAdmin == true;  // Admins can read/write to anywhere.
```

Comment:
@nrimsky, I didn't realise we could do this, but it does address our concerns about restricting admin permissions!